### DISCUSSION TITLE
Welcome to indi-allsky Discussions!

### DISCUSSION THREAD

**Discussion by aaronwmorris** [aaronwmorris/indi-allsky#341] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by GregHD74** [comment]
HI everybody, I very new to this Raspberry stuff so I might have a lot of questions.All I want to do is use an all sky cam to look at sky conditions. When I opened Indi it asks for a username and password. I do not recall putting anything in when setting up. Is there a default set that would work? And what might that be?
Thanks Greg

**Reply by aaronwmorris** [reply]
There are no default credentials for indi-allsky.  You will have to reset the password of an existing user or create a new user from the CLI.

https://github.com/aaronwmorris/indi-allsky/wiki/Web-Interface-Password

**Reply by GregHD74** [reply]
ok, Thank You
Greg

**Comment by NC48401** [comment]
Hi Aaron. New to the site and just getting my indi-allsky running. Are there any tutorials around for things like posting to a remote website? I am illiterate when it comes to the Pi and Linux so struggling a bit as I learn. I do have TJ's running on my website.

**Reply by morganjrees** [reply]
could do with a tutorial of any sorts if anyone has one or wants to make one?

**Reply by WistowTBone** [reply]
I had a real fun time getting my software up and operational but used AI to walk me through it. took a lot of faffing but main issue turned out to be certificates for web service.

**Comment by morganjrees** [comment]
Hi All just setup my Indi-allsky today, happy to contribute ton any hardware compatibly or testing needed with my setup. RPi 3B, Axis IP Camera. I’m a ham radio operator so looking forward to testing the ADSB integration.

**Comment by KJ5IKBob** [comment]
Hello everyone,

When attempting to install indi-allsky I keep running into this message:

Please run ./miscindi.sh before running ./setup.sh 

I've run it as instructed, but when I attempt to run ./setup.sh again the message keeps popping up. What am I doing wrong?

Running Debian Bookworm on a Pi 4 B with 8 GB RAM and 256 GB mini SD card.

**Reply by aaronwmorris** [reply]
I am assuming it wants you to run `./misc/build_indi.sh`  This script should require about 40-50 minutes to build the INDI library on a Pi4.  Are you sure the script ran to completion?

**Comment by KJ5IKBob** [comment]
Well, if it should take 40-50 minutes, I don't know. When I run it there is a bunch of stuff scrolling on the screen, which I expect, and then it stops and I get the command line prompt. This process takes about 30 seconds. Just now I have tried to run it again, and this time it appears stuck on the screen where it says < Ok > on the bottom. I have pressed the ENTER key, but nothing seems to be happening. I have turned the Pi off and back on after sending the first message.

**Reply by aaronwmorris** [reply]
I would need to see the messages being generated by the script.

**Reply by KJ5IKBob** [reply]
Attached.
![20250801_194856](https://github.com/user-attachments/assets/eb099aaa-d74a-4401-b7e9-7aa2f5a74500)
![20250801_194902](https://github.com/user-attachments/assets/45800852-93e9-49f2-8154-3fd59156ecd8)

**Reply by aaronwmorris** [reply]
I have seen issues like this in the past.  I would try using a different DNS server and maybe that will resolve it.

```
# change to 8.8.8.8 google dns
echo "nameserver 8.8.8.8" | sudo tee /etc/resolv.conf

# clear apt caches
sudo apt-get autoclean
```

Rerun `./misc/build_indi.sh`

**Reply by KJ5IKBob** [reply]
Slightly different result, but still less than 30 seconds.

I appreciate your time, by the way.

![20250801_205028](https://github.com/user-attachments/assets/112e8c88-5c19-44d1-8b17-b7df035cac0c)
![20250801_205042](https://github.com/user-attachments/assets/b8214722-d45c-4820-b8af-255fa2051342)

**Reply by aaronwmorris** [reply]
Just keep running it.  You may try rebooting your Raspi.  Perhaps move the Pi closer to the router if it is connected via wifi.  Maybe even try rebooting your home internet router.

**Reply by KJ5IKBob** [reply]
I'll try all those things. The Pi is using ethernet. Again, appreciate the help!

**Reply by KJ5IKBob** [reply]
Well, I gotta say I would've thought the ethernet connection was more stable than wifi, but I pulled that plug and hooked up to wifi and that thing just went to town! Thanks again for your time!

Regards,

Bob

**Reply by aaronwmorris** [reply]
hah, I do not think I would have predicted that either!  Perhaps it is a bad ethernet cable. Glad to help.

**Comment by rayscrim** [comment]
Hello Aaron and Team!

Newbie here getting used to indi/Pi/Linux/Python/etc. First I want to say what a fantastic job you have done on this. I can't even imagine the effort that has gone into making it just 'work'.

I have a Pi 4 that was collecting dust and decided to use it for an allsky setup. Had it working with the Raspberry HQ camera for a few weeks and after the camera died I decided to do a couple of upgrades. I now have an ASI 676 and am trying to change the way I control the dew heater (by using the Pi vs. using temp sensors on the dew heater - Kendrick controller). I am trying to use a DHT11 sensor to control a relay for the heater, but cannot for the life of me get allsky to 'recognize' the sensor. I had allsky reading a DS18B20 and have also been able to read the DHT11 with HiPi.io's code. My thoughts of using the DHT11 vs. the DS18B20 were that I live in a humid environment and may find that I need to control the dew heater based on humidity, but I'm not even sure about this... Note I am using OpenWeather on Sensor A slots 10-19 and can also get weather info that way, but would still want or need to know the enclosure temperature.

This is my first github post and first question so I'm not even sure what to post, where to get info, whether this question has been asked etc. so bear with me. I can say I have the DHT11 on GPIO4 (which HiPi.io has read successfully) using Sensor B and slots 20-22, and for note I have the relay on GPIO17. Note the GPIO's are what I set in the Config 'Sensor' and 'Devices' tabs.

Other notes are that I get errors when running dht11_22_blinka.py, tempSensorDht.py and sensor_test.py. I think I have included txt files with the errors.

Any help or guidance is greatly appreciated!
Thank you in advance.
Ray

[dht11_22_blinka_error.txt](https://github.com/user-attachments/files/24303780/dht11_22_blinka_error.txt)
[tempSensorDht_error.txt](https://github.com/user-attachments/files/24303781/tempSensorDht_error.txt)
[sensor_test_error.txt](https://github.com/user-attachments/files/24303836/sensor_test_error.txt)

**Reply by aaronwmorris** [reply]
The `GPIO Busy` message means another process is using the GPIO pin.  Are you are indi-allsky is not running in the background?  That would cause the GPIO pin to become locked.  The sensor test cannot be run while indi-allsky is running.

**Reply by rayscrim** [reply]
Thank you for your quick reply!

Oops yes indi-allsky was running <embarrasing>. By the time I re-ran sensor_test I had reinstalled the DS18B20. Sensor_test indicated that the 1 wire interface was not enabled, so I enabled it, rebooted, and re-ran sensor_test. All good with the DS18B20. 

I swapped back the DHT11, reconfigured the Sensor B in Config and Admin tabs and rebooted. At that point I re-ran sensor_test with the DHT11 and got the same series of errors attached (indi-allsky not running). After running the test, I restarted indi-allsky to see if the sensor was recoginzed but it wasn't (and my relay was not turning on). The dew heater expected status was High (relay on) with the thresholds I'm using, which worked with the DS18B20. Note this is my 2nd DHT11 sensor and I do have 2 more new spares, but it seems as though something in the sensor_test result may lead a clue. I'm not sure how to decipher the results though. It may be more trouble than it's worth, since I can seem to get things working with the DS18B20 sensor...

[sensor_test_error2.txt](https://github.com/user-attachments/files/24321089/sensor_test_error2.txt)

**Reply by aaronwmorris** [reply]
Make sure you use a different pin for the DHT11 than the one you use for the DS18B20.  Once the 1-wire interface is active, the 1-wire pin cannot be use for GPIO purposes.

**Comment by rayscrim** [comment]
Ok just making sure I understand this one. Most online documentation showed GPIO4 for a DHT11. To clarify I have removed the DS18B20 and the only other sensor/device I have is the relay on GPIO17. This is the setup that worked for both the relay and DS18B20. 

I'm writing/testing at the same time. I disabled the 1 wire interface, kept GPIO4 for DHT11 and GPIO17 for the relay. Neither the relay, nor the DHT11 are working (Open Weather is working FYI).

For note, I noticed I2C is enabled in Pi Config, not sure if this is worth mentioning. I left it as is.

Next test wass to disable 1 wire interface, move DHT11 from GPIO4 to GPIO27. DHT11 and relay (on GPIO17) did not work.

Next I did the same with 1 wire enabled. DHT11 and relay did not work.

Next test was to diable I2C (1 wire still enabled). I also turned ON use_pulseio (which was previously turned off). This configuration got the DTH11 WORKING!!! Since I changed multiple things at once, I tried to see which setting enabled the DHT11. It seems as though the use_pulseio was the last piece as the DHT11 goes back to all zeros with it off. 

Now my relay is not working... I tried re-enabling I2C but this did not work. I'll keep trying. Maybe a different pin.

Thank you for your help Aaron!!

**Reply by rayscrim** [reply]
Update!  I did not have Enable Daytime checked and the relay seems to be working now. It has always been what I call sluggish and it seems saving configuration changes take longer with the relay enabled (seems to hang on Reloading). Not sure if related, but any changes (such as inverting output to check relay on/off) seem to need rebooting to take effect. That may be what's needed anyway so I'll go with that. Going to try the setup outside now...

**Reply by aaronwmorris** [reply]
Adding the Fan config might add a few seconds to the reload time, but it should not be excessive.

**Reply by rayscrim** [reply]
Another update!
System is working. The Reload time is longer, and seems to be getting less though for whatever reason. I shouldn't need to reload much once I'm set up.  I was able to get the camera in the housing and did some temporary wiring to debug focus, exposure, orientation, etc. And of course first light of the second system was of course (attached).

[ccd2_20251224_154555](https://github.com/user-attachments/assets/54e741f7-a0b0-4639-95f1-31356f2c2035)

Clouds!!

I'll tweak over the next few days and hopefully we get some clear spots to dial in the focus. 

Again thank you Aaron for your help!

**Comment by jensdk3** [comment]
Hi all and happy New Year to all! 
I wonder why the link to startrails movie in the Media->Time Lapses section do not work anymore. It used to work until recently when I made changes to movie and overlay settings. I can see that the startrail mp4 movie is produced and put in the expected folder. The link to the Keogram movie is also still working. I have rebooted the raspberry 5. Any ideas for a fix? 
BR
Jens

**Comment by camthegit** [comment]
I am getting video worker restart failure errors. One problem seems to be that the following URL no longer exists. 
>  URL returned 404
> 2026-07-04T15:36:10.036883+10:00 skycam [WARNING] Video-14-48518/MainThread aurora.download_json() [557]: Downloading ht
> tps://services.swpc.noaa.gov/products/solar-wind/mag-2-hour.json. 
I wonder if there might also be a problem dealing with absent data - I am not in an area where Auroras tend to be visible.  Can I turn the aurora scripting off? Although I would prefer to fix it.
Thanks for a great program

**Reply by aaronwmorris** [reply]
The NOAA Space Weather Center recently changed instruments for solar wind data.  Unfortunately, they changed the URLs for the data (and changed formats as well).  This was fixed in #2963 

https://www.spaceweather.gov/news/solar-wind-data-and-display-changes

**Reply by camthegit** [reply]
Great - thanks for the response!
